src/organisms/Status/collections/MissingPermissions.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import { useNavigate } from 'react-router';
+
+import { Status } from 'src/organisms/Status';
+
+export const MissingPermissions: FC = () => {
+  const navigate = useNavigate();
+
+  const handleOnClick = () => {
+    navigate(-1);
+  };
+
+  return (
+    <Status>
+      <Status.Title title="It looks like you don't have permission to access this page." />
+      <Status.Action onClick={handleOnClick} />
+    </Status>
```

Comment:
On the error page we had the possibility to add the list of items you need to apply to in access it. Like for inpress it could be, SMDA, PDM, and SSDL. 
Did we remove that premade component with the status page?

Reply:
Yeah it's has been removed, but I can add it back